src/markdown/index.ts

```diff
@@ -133,3 +134,9 @@ export function markdownToHTML(markdown: string, options: TMarkdownItOptions = {
     return '';
   }
 }
+
+export const applyMathpixMarkdownPlugins = (options: TMarkdownItOptions = {}): MarkdownIt => {
+  let md: MarkdownIt = mdInit(options);
+  md = injectRenderRules(md);
```

Comment:
Maybe it could better to change `injectRenderRules` to infer the type for MarkdownIt
`injectRenderRules = (renderer: MarkdownIt) => {`